README.md

```diff
@@ -115,174 +80,60 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+GETTING STARTED 
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
+A new develpoer can use this project to learn how to build a portfolio.
 To get a local copy up and running, follow these steps.
 
-### Prerequisites
-
+   git clone git@github.com/david30-maker/finish-mobile-version.git
+Prerequisites
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
-
-### Setup
-
+VSCode editor
+Browser
+Setup
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
-
-### Install
-
-Install this project with:
-
-<!--
-Example command:
+  git clone git@github.com/david30-maker/finish-mobile-version.git
+  cd Mobile-first-project
 
-```sh
-  cd my-project
-  gem install
-```
---->
+Install:
+Install project dependencies in this project with:
 
-### Usage
+  npm install
 
-To run the project, execute the following command:
+Usage:
+To run the project, open the project with live server and visit http://localhost:5500
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
-
-```
- -->
+Open in browser
+http://localhost:5500
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
+## 👥 Author
 
 > Mention all of the collaborators of this project.
 
-👤 **Author1**
+👤 **David Igbo**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@david30-maker](https://github.com/githubhandle)
+- Twitter: [@davidigbo](https://twitter.com/twitterhandle)
+- LinkedIn: [linkedin.com/in/dvid-igbo-464196184/]](https://linkedin.com/in/linkedinhandle)
 
 👤 **Author2**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
 
-## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
+ Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- [ ] **contact form**
+- [ ] ****
+- [ ] 
 
```

Comment:
- [ ] Excellent job adding the `sections/headers`. Let's make sure that all the sections are present in README. Right now the sections after the `Authors` are missing. Don't worry just a more more step to go 💪 Please make sure you add them

### **_Your README now 👇_**

![image](https://user-images.githubusercontent.com/106119105/213111810-1b6443ba-0b74-4c4d-95bd-064f61384848.png)